spec.html

```diff
@@ -115,6 +115,27 @@ <h1>Tree Grammar</h1>
   attribute boolean isCaptured;
 };
 
+interface AssertedPositionalParameterName {
+  attribute unsigned short index;
```

Comment:
I feel like this should be `unsigned long`. AFAIK there's no normative limit on the number of parameters, but given that `arguments` is [array indexed](https://tc39.github.io/ecma262/#array-index), `unsigned long` seems okay.